app/controllers/bookings_controller.rb

```diff
@@ -32,8 +32,16 @@ def update
     @booking = Booking.find(params[:id])
     @booking.update(booking_params)
     authorize @booking
```

Comment:
```suggestion
    # authorize @booking
```
We can add authorize in the respond, so that we don't need to repeat it twice